tests/bundler.js

```diff
@@ -167,6 +179,39 @@ describe( 'bundler', () => {
 	} );
 } );
 
+describe( 'bundler/jsonPackage', () => {
+	before( () => {
+		rimraf( 'tests/fixtures/jsonPackage/dist' );
+	} );
+
+	after( () => {
+		rimraf( 'tests/fixtures/jsonPackage/dist' );
+	} );
+
+	// #155
+	it( 'should load JSON file', () => {
+
+		function testExecution( filename ) {
+			const codeES5 = readFileSync( filename ).toString();
+			const consoleSpy = spy( console, 'log' );
+
+			// I'm so sorry!
+			eval( codeES5 );
```

Comment:
I'm not sure if there is a need to evaluate the bundled file. In fact it checks if `@rollup/plugin-json` works correctly, not the bundler. From the bundler perspective the only issue is the error being emitted.

But if we're going to also check the bundled code, I'd be for two things:

*   add also a test to see if there is no error emitted,
*   replace `eval` with [`execFile`](https://nodejs.org/api/child_process.html#child_process_child_process_execfile_file_args_options_callback) – it seems much more elegant.